symbology/simple-fill-symbol/src/main/java/com/esri/samples/simple_fill_symbol/SimpleFillSymbolSample.java

```diff
@@ -75,14 +70,43 @@ public void start(Stage stage) {
       ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
 
       // create a control panel
-      controlsVBox = new VBox(6);
+      var controlsVBox = new VBox(6);
       controlsVBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0,0,0,0.3)"), CornerRadii.EMPTY,
-          Insets.EMPTY)));
+        Insets.EMPTY)));
       controlsVBox.setPadding(new Insets(10.0));
       controlsVBox.setMaxSize(180, 200);
       controlsVBox.getStyleClass().add("panel-region");
 
-      createSymbolFunctionality();
+      // create a fill symbol with an outline
+      var fillSymbol = new SimpleFillSymbol(Style.SOLID, Color.RED, new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, Color.BLUE, 3));
+
+      // create a color picker that updates the fill symbol's color property
+      var fillLabel = new Label("Change Fill Color");
+      fillLabel.getStyleClass().add("panel-label");
+      var fillColorPicker = new ColorPicker();
+      fillColorPicker.setMaxWidth(Double.MAX_VALUE);
+      fillColorPicker.valueProperty().bindBidirectional(fillSymbol.colorProperty());
+
+      // create a color picker that updates the fill symbol's outline color property
+      var outlineLabel = new Label("Change Outline Color");
+      outlineLabel.getStyleClass().add("panel-label");
+      var outlineColorPicker = new ColorPicker();
+      outlineColorPicker.setMaxWidth(Double.MAX_VALUE);
+      outlineColorPicker.valueProperty().bindBidirectional(fillSymbol.getOutline().colorProperty());
+
+      // create a combobox that updates the fill symbol's fill style
+      var styleLabel = new Label("Change Fill Style");
+      styleLabel.getStyleClass().add("panel-label");
+      ComboBox<Style> styleBox = new ComboBox<>();
+      styleBox.getItems().addAll(Style.BACKWARD_DIAGONAL, Style.FORWARD_DIAGONAL, Style.DIAGONAL_CROSS, Style.HORIZONTAL,
```

Comment:
If we don't care about the order we could future proof with 
`styleBox.getItems().setAll(Arrays.asList(Style.values()));`

Reply:
Ordinarily this would be a better approach but I'll leave it as it's easier for someone to grab the actual style Enums with them written out explicitly. So it's a bit of a Sample-ism this one.